gcc_test_script.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+
+set -e
+
+usage() {
+  cat <<EOF
+  usage: $0
+  This script fetches and runs available test suites using runners
+  provided in ./*-tests-runner dirs.
```

Comment:
Please do not use abbreviations when informing the user about anything, errors, help, warnings, etc... (dirs -> directories).